docs/src/main/paradox/routing-dsl/directives/attribute-directives/attribute.md

```diff
@@ -0,0 +1,27 @@
+# attribute
+
+@@@ div { .group-scala }
+
+## Signature
+
+@@signature [AttributeDirectives.scala](/akka-http/src/main/scala/akka/http/scaladsl/server/directives/AttributeDirectives.scala) { #attribute }
+
+@@@
+
+## Description
+
+Extracts the value of the request attribute with the given key.
```

Comment:
Link to the [Attribute docs](https://doc.akka.io//docs/akka-http/snapshot/common/http-model.html#attributes) for more context. Possibly explain how the attribute would get there before hitting the routing DSL.